scripts/pi-hole/php/savesettings.php

```diff
@@ -452,6 +452,19 @@ function readAdlists()
 				{
 					exec('sudo pihole -a -c');
 				}
+				$adminemail = trim($_POST["adminemail"])
```

Comment:
adding a ```;``` on the end of this statement would stop the parse error that stops the settings page from loading.
Would also probably be a good idea to do the same on lines 458 and 462

Reply:
Fixed. Sorry about that, habit from working in other languages